modules/300-prometheus/docs/FAQ.md

```diff
@@ -260,10 +260,12 @@ data:
   prometheus.conf.template: |
     server {
       listen 80 default_server;
+      resolver kube-dns.kube-system.svc.cluster.local valid=30s;
```

Comment:
Cluster domain may differ for Deckhouse installations. Can we avoid providing the full domain name somehow?

Reply:
We don't have the right way to do this. Issue about downward api - https://github.com/kubernetes/dns/issues/421.
So i added manual fill of cluster domain to yaml file.